src/main/java/tech/gruppone/stalkerserver/configuration/InfluxDBPoint.java

```diff
@@ -0,0 +1,65 @@
+package tech.gruppone.stalkerserver.configuration;
+
+import java.time.Instant;
+import org.influxdb.annotation.*;
+import jdk.jfr.Timestamp;
+
+@Measurement(name = "accessLog")
+public class InfluxDBPoint{
```

Comment:
va spostata in nel package tech.gruppone.stalkerserver.configuration e la classe va rinominata in LocationUpdateTimeSeriesRepository